### PR TITLE
(maint) Minor docs updates on running litmus tests

### PR DESCRIPTION
**Pull request by pmcmaw**

### CODE REVIEW

File: Running-acceptance-tests.md [hunk 1]
```diff
@@ -231,6 +242,7 @@ The MoTD shows you how to use Litmus to acceptance test an existing module. As y
 
 * Provision more than one system, for example, `pdk bundle exec rake 'litmus:provision[docker, centos:6]'`. Note that you will need to re-run the `install_agent` and `install_module` command if you want to run tests.
 * Look at the inventory file and take note of the ssh connection information
-* ssh into the CentOS box, for example, `ssh -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no root@localhost -p 2222`, or use Bolt as shown in the example.
+* ssh into the CentOS box when you know the password, for example, `ssh -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no root@localhost -p 2222`, or use Bolt as shown in the example.
+* ssh into the CentOS box without a password, run `docker ps`, take note of the Container ID and then run `docker exec -it bf5c9c035360 '/bin/bash'` in this example bf5c9c035360 is the Container ID.
```
Comment (ghoneycutt) [inline Running-acceptance-tests.md:246]: The names might be a bit easier such as `litmusimage_centos_7-2222`

File: Running-acceptance-tests.md [hunk 2]
```diff
@@ -133,6 +133,13 @@ Successful on 1 target: localhost:2222
...
 ```
 
+If you want to install a specific version of puppet you can use the following command:
+```
+pdk bundle exec rake 'litmus:install_agent[puppet6]
+```
+Examples of other versions you can pass in are: puppet6-nightly, puppet7, puppet7-nightly.
+
+
 ## 5. Install the MoTD module on the CentOS image.
```
Comment (ghoneycutt) [inline Running-acceptance-tests.md:143]: Suggest adding something here like the following as it was not immediately apparent that this step was needed every time code changes.

```
If you are interactively modifying code and testing, note that this step must be run after changing the code before the acceptance tests are run.
```

File: Running-acceptance-tests.md [hunk 3]
```diff
@@ -141,11 +148,17 @@ To install the MoTD module on the CentOS image, run the following command from i
 pdk bundle exec rake litmus:install_module
 ```
 
-You see output similar to:
+*Note: If you are interactively modifying code and testing, this step must be ran after your changes are made and before you run your tests.*
```
Comment (sanfrancrisko) [inline Running-acceptance-tests.md:151]:
```suggestion
*Note: If you are interactively modifying code and testing, this step must be run after your changes are made and before you run your tests.*
```

**Review by sanfrancrisko — COMMENTED** [review]
Just one minor comment

**Pull request merged**